Cargo.lock

```diff
@@ -539,6 +547,17 @@ dependencies = [
 ]
 
 [[package]]
+name = "sdl2_ttf"
+version = "0.9.0"
+source = "git+https://github.com/cristicbz/rust-sdl2_ttf?branch=wrapped#ce360f9445391074a41ca0ec6b27082d07abf52c"
```

Comment:
I can't build anymore because of this message:

```
$ cargo build -j4 --verbose
    Updating git repository `https://github.com/cristicbz/rust-sdl2_ttf`
Unable to update https://github.com/cristicbz/rust-sdl2_ttf?branch=wrapped#ce360f94

Caused by:
  [4] Revspec 'ce360f9445391074a41ca0ec6b27082d07abf52c' not found.
```

Any idea of why?